TAM_SRC/BT.cpp

```diff
@@ -36,31 +36,38 @@ void WordSwap(char *str){
     }
     temp[j][n]='\0';
 
-    for(i=0; i<=countspace(str); i++){
-        if(i==1)
-            cout << temp[countspace(str)-1] << " ";
-        else
-            if(i==countspace(str)-1)
+    for(i=0; i<=CountSpace(str); i++){
+        if(i==1){
+            cout << temp[CountSpace(str)-1] << " ";
+        }
+        else{
+            if(i==CountSpace(str)-1){
                 cout << temp[1] << " ";
-            else
-                if(i==countspace(str))
+            }
+            else{
+                if(i==CountSpace(str)){
                     cout << temp[i];
-                else
+                }
+                else{
                     cout << temp[i] << " ";
+                }
+            }
+        }
     }
 }
 
-list<int> findAbsence(int arr[], int size){
+list<int> FindAbsence(int arr[], int size){
```

Comment:
Can you explain why you use list here? Can we simply use normal array?

Reply:
if you use normal array, you need to determine size of array. But in this case, it can't determine because i don't know exactly how many student absence. But if i use list i do not care how many student absence, i only use function add. If list was full it auto increase. It easy use !

Reply:
if i use normal array to solve problem this. it very complex and source code will long. but list or classes in Containers of C++ it will simple.